src/css/modules/empty.css

```
@@ -0,0 +1,46 @@
+  text-align: center;
+  box-sizing: border-box;
+}
+.lay-empty-icon {
+  margin-bottom: var(--lay-empty-icon-gap);
+  font-size: var(--lay-empty-icon-size);
+  line-height: 1;
+  color: var(--lay-gray-500);
+}
+/* 内置默认图标：文件图标。容器为空时自动显示，放入内容后自动隐藏 */
+.lay-empty-icon:empty::before {
+  font-family: 'lay-icon' !important;
```

Stylelint (17.14.0)

[error] Expected no quotes around "lay-icon" (font-family-name-quotes)

(font-family-name-quotes)